components/style/properties/data.py

```diff
@@ -128,13 +128,6 @@ def new_style_struct(self, *args, **kwargs):
     def active_style_structs(self):
         return [s for s in self.style_structs if s.additional_methods or s.longhands]
 
-    def switch_to_style_struct(self, name):
-        for style_struct in self.style_structs:
-            if style_struct.trait_name == name:
-                self.current_style_struct = style_struct
-                return
-        raise Exception("Failed to find the struct named " + name)
-
```

Comment:
Since all style structs are now being entirely contained per-file, we no longer need this helper method.